### PR TITLE
feat(button): stacked style flow support

### LINKED ISSUE #1156: [Button] add stacked layout
image

### PR DESCRIPTION
**Pull request by yinonov**
add button stacked flow style to meet teams demand

Closes #1156

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: components/button/src/vwc-button.scss [hunk 1]
```diff
@@ -117,6 +105,32 @@
 	:host([enlarged]) & {
 		--mdc-button-horizontal-padding: 20px;
 	}
+
+	:host([stacked][enlarged]) & {
```
Comment (yinonov) [inline components/button/src/vwc-button.scss:109]: we can leverage the `button-stacked` class here
with sass
```css
.button-stacked {
    :host([enlarged]) & {

    }
}
```

**Comment by yinonov** [comment]
Ripple overflows the button region

image

**Comment by rachelbt** [comment]
> Ripple overflows the button region
> 
> image
> 
> (Happens on my iOS)

it happens also in Safari for all the outlined+pill buttons. I'll check it out

**Comment by yinonov** [comment]
> > Ripple overflows the button region
> 
> > 
> 
> > image
> 
> > 
> 
> > (Happens on my iOS)
> 
> 
> 
> it happens also in Safari for all the outlined+pill buttons. I'll check it out. Maybe we should deal with it in a separate issue?

If it's not related to this new change, of course

File: components/button/src/vwc-button.scss [hunk 2]
```diff
@@ -10,9 +9,9 @@
 	$default: primary,
 );
 @use '@vonage/vvd-foundation/scss/mixins/connotation' as connotation;
-@use '@vonage/vvd-foundation/scss/mixins/shape-mixins' with(
-	$variable-name: --vvd-button-shape
-);
+@use '@vonage/vvd-foundation/scss/mixins/shape-mixins'with (
```
Comment (yinonov) [inline components/button/src/vwc-button.scss:12]:
```suggestion
@use '@vonage/vvd-foundation/scss/mixins/shape-mixins' with (
```

File: components/button/src/vwc-button.scss [hunk 3]
```diff
@@ -10,9 +9,9 @@
...
+	$variable-name: --vvd-button-shape)
+;
```
Comment (yinonov) [inline components/button/src/vwc-button.scss:14]:
```suggestion
	$variable-name: --vvd-button-shape
);
```

**Pull request merged**